### ISSUE TITLE
Issue with Redirection plugin

### ISSUE THREAD

**Issue by marcselman** [faktorvier/f4-error-pages#1]
When using F4 Error Pages together with the Redirection plugin from John Godley, the 404 logging feature no longer works.

Normally every 404 error would be logged by the Redirection plugin but when F4 Error Pages is active the logging no longer works.

**Comment by Stibo** [comment]
Hi marcselman

Thanks for the feedback. We'll have a look at this and try to add compatibility with the Redirection plugin if possible.

**Comment by marcselman** [comment]
Hi @Stibo 
Thanks. Could it be that your plugin prevents catches the 404 event so other plugins that also react tio that event don't receive it anymore?
Thanks for taking the time! It's a nice plugin and pretty unique. For some reason there are a lot of plugins to redirect 404 pages to the homepage, but that's a bad practice and not the purpose of a 404 page ofcourse 😄.

**Comment by Stibo** [comment]
Hi @marcselman 

I agree with you, redirecting back to the home page is not the best practice for that, thats why we developed this little plugin. We're glad you find this plugin helpful too!

You are right, there was a timing problem with our function that is triggered shortly before the one that should track the 404 page in the Redirection plugin. So the Redirection function was not triggered anymore. We've moved our function after this one so every 404 should be tracked correctly.

We've released a new version for the plugin and it should work as expected now. If you have any other troubles, don't hesitate to come back at us.

Cheers!
